packages/core/src/components/DataTable/columns/renderDataColumns.tsx

```diff
@@ -43,17 +43,15 @@ export default function renderDataColumns(
     expandable,
   }: ArgumentsFromProps,
 ) {
-  const renderCell = (key: string, isLeftmost: boolean) => (row: TableRow) => {
+  const renderCell = (key: string, isLeftmost: boolean) => (row: VirtualRow) => {
     const { metadata } = row.rowData;
     const { isChild } = metadata;
     const customRenderer = renderers && renderers[key];
-
     const indentSize = !expandable || !isLeftmost ? 2 : 2.5;
     const spacing = isChild || !((expandable || selectable) && isLeftmost) ? indentSize : 0;
-
     const rendererArguments: RendererProps = {
       row,
-      key,
+      keyName: key,
```

Comment:
@schillerk Breaking?

Reply:
Looks like it's only being used in Bighead, not a big deal